aws-s3-rm: add page (#9218)
tldr-pages/tldr: pages/common/aws-s3-rm.md

# aws s3 rm

> > Delete an S3 object.

Reviewer: ```suggestion
> Delete S3 objects.
```

> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/rm.html>.

> - Delete a single S3 object:

Reviewer: ```suggestion
- Delete a specific S3 object:
```

> `aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt`

Reviewer: ```suggestion
`aws s3 rm s3://{{bucket_name}}/{{path/to/file}}`
```

> - Perform a dry-run of a single S3 object deletion:

Reviewer: ```suggestion
- Preview the deletion of a specific S3 object without delete it (dry-run):
```

> `aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt --dryrun`

Reviewer: ```suggestion
`aws s3 rm s3://{{bucket_name}}/{{path/to/file}} --dryrun`
```

> - Delete an object from an S3 access point:

Reviewer: ```suggestion
- Delete an object from a specific S3 access point:
```

> `aws s3 rm s3://arn:aws:s3:us-west-2:123456789012:{{access_point}}/{{my_access_point}}/{{my_key}}`

Reviewer: I think this is the correct placeholders
```suggestion
`aws s3 rm s3://arn:aws:s3:{{region}}:123456789012:access_point/{{access_point_name}}/{{object_key}}`
```

Author: @navarroaxel I've updated it to look like this example from the AWS user guide [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html)

```
arn:aws:s3:region:account-id:accesspoint/access-point-name/object/resource
```

Let me know if there shouldn't be curly braces around the `account_id` and `access_point` values!

Thanks for pointing this out
